evm/db/chain.py

```diff
@@ -250,15 +255,28 @@ def get_block_header_by_hash(self, block_hash: Hash32) -> BlockHeader:
                 encode_hex(block_hash)))
         return _decode_block_header(header_rlp)
 
-    def header_exists(self, block_hash: Hash32) -> bool:
-        """Returns True if the header with the given block hash is in our DB."""
-        return self.db.exists(block_hash)
+    def get_block_uncles(self, uncles_hash: Hash32) -> List[BlockHeader]:
+        """
+        Returns an iterable of uncle headers specified by the given uncles_hash
+        """
+        validate_word(uncles_hash, title="Uncles Hash")
+        return rlp.decode(self.db.get(uncles_hash), sedes=rlp.sedes.CountableList(BlockHeader))
+
+    def get_score(self, block_hash: Hash32) -> int:
+        """
+        Returns the score for the header with the given hash.
+
+        Raises HeaderNotFound if no header with the given has is found in the database.
```

Comment:
Looks like this is missing the `HeaderNotFound` raise, which I suppose means that it's not tested.